bugsnag/event.py

```diff
@@ -124,6 +124,12 @@ def get_config(key):
         for name, tab in options.items():
             self.add_tab(name, tab)
 
+        if hasattr(exception, "__notes__"):
+            self.add_tab(
+                "exception notes",
+                dict(enumerate(exception.__notes__))
```

Comment:
The lint error for the line being too long covered up a mypy error:

> bugsnag/event.py:130: error: "BaseException" has no attribute "__notes__"

It's fine to just ignore it since we're checking with `hasattr` immediately beforehand:

```suggestion
                dict(enumerate(exception.__notes__)) # type: ignore # noqa
```